internal/demo/util/worker.go

```diff
@@ -33,11 +33,11 @@ func NewVersionedWorker(opts worker.Options) (w worker.Worker, stopFunc func())
 
 	opts.DeploymentOptions = worker.DeploymentOptions{
 		UseVersioning:             true,
-		Version:                   mustGetEnv("TEMPORAL_DEPLOYMENT_NAME") + VersionIDSeparator + mustGetEnv("WORKER_BUILD_ID"),
+		Version:                   mustGetEnv("TEMPORAL_DEPLOYMENT_NAME") + VersionIDSeparator + mustGetEnv("TEMPORAL_WORKER_BUILD_ID"),
```

Comment:
did the SDK upgrade PR not land? I'd expect this to be `worker.WorkerDeploymentVersion` not a string